index.html

```diff
@@ -49,180 +49,226 @@ <h2 class="hex connect">LET’S CONNECT</h2>
-                <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
-                </li>
-                <li class="lang">
-                <img src="images/langusges/js.png" alt="js" />
-                <p>JavaSript</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/html.png" alt="js" />
-                <p>HTML</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/css.png" alt="js" />
-                <p>CSS</p>
-                </li>
-                <li class="frameworks">
-                <p class="langusge">Frameworks</p>
-                <p class="sign">></p>
-                </li>
-                
-                <li class="frameworks">
-                <p class="langusge">Skills</p>
-                <p class="sign">></p>
-                </li>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/js.png" alt="js" />
+                        <p>JavaSript</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/html.png" alt="js" />
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/css.png" alt="js" />
+                        <p>CSS</p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Frameworks</p>
+                        <p class="sign"></p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Skills</p>
+                        <p class="sign"></p>
+                    </li>
                 </ul>
-                </div>
             </div>
-
+        <section class="contact-me"> 
+            <div class="contact-background">
+                <h2>Contact me</h2>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need to build or a project that needs coding. I'd love to help with
+                </p>           
+                <form>
+                    <input type="text" id="fname" name="fname" placeholder="lorem Ipsum" maxlength="30" required><br>
+                    <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required><br>
+                    <textarea name="message" id="message" maxlength="500" required>Write your message here...</textarea><br>
+                    <div class="btn-submitContact">
+                      <button type="submit" id="submit" name="submit" value="submit">Get In Touch</button>
+                    </div>
```

Comment:
- [ ] Dear @Jonathan-kiprono, For the submit button we need to follow interactions such as hover, active (when it's pressed), disable, and enable. This is to make sure the user has a pleasant time surfing our page. 

### Find this in the figma template, please.
![image](https://user-images.githubusercontent.com/69077061/230015517-4b802f5f-df32-41a2-a423-5bc3679852be.png)